src/consensus/default_constants.py

```diff
@@ -49,6 +49,7 @@
     # Max block cost in clvm cost units (MAX_BLOCK_COST * CLVM_COST_RATIO_CONSTANT)
     "MAX_BLOCK_COST_CLVM": 225359496,
     "WEIGHT_PROOF_THRESHOLD": 2,
+    "SUB_BLOCKS_CACHE_SIZE": 10000,
```

Comment:
Let's set this to EPOCH_SUB_BLOCKS + 3*MAX_SUB_SLOT_SUB_BLOCKS ?  and add a comment so we don't forget